src/main/java/TaskList.java

```diff
@@ -0,0 +1,59 @@
+public class TaskList {
+
+    private static Task[] tasks = new Task[100];
+    private static int taskCount = 0;
+    private static final String LINE_DIVIDER = "-------------------------------------";
+    private static final String ROGER_MESSAGE = "Got it. I have added this task:";
+
+    private static void printRoger(Task task) {
+        System.out.println(ROGER_MESSAGE);
+        System.out.println(task);
+        System.out.println("You now have " + taskCount + " tasks.");
+        System.out.println(LINE_DIVIDER);
+    }
+
+    private static void printMark(int target, boolean mark) {
+        if (mark) {
+            System.out.println("Marked this task: \n " + tasks[target - 1]);
+        } else {
+            System.out.println("Unmarked this task: \n" + tasks[target - 1]);
+        }
+        System.out.println(LINE_DIVIDER);
```

Comment:
Phrases "Marked this task: \n" and "Unmarked this task: \n" can be written as a constant String when outputting
 like line 21 declared in line 5